### ISSUE TITLE
"background.service_worker is currently disabled" in Firefox

### ISSUE THREAD

**Issue by Jipok** [kevmo314/magic-copy#20]
I get zip from releases
![screenshot](https://user-images.githubusercontent.com/25588359/232198231-2171bbc0-3512-4a1a-9ac9-e2dd9940c421.png)

**Comment by kevmo314** [comment]
It seems that we have to use a separate manifest file for Firefox because it doesn't support `background.service_worker` in Manifest V3 yet :(

**Comment by Jipok** [comment]
So this add-on is only for chrome?

**Comment by kevmo314** [comment]
> So this add-on is only for chrome?

At the moment, yes. You can try the Firefox release but I haven't been able to test it thoroughly.

**Comment by SputNikPlop** [comment]
> It seems that we have to use a separate manifest file for Firefox because it doesn't support `background.service_worker` in Manifest V3 yet :(

Yeah looks like https://github.com/mozilla/web-ext/issues/2532 but there seems to be progress based on https://bugzilla.mozilla.org/show_bug.cgi?id=1573659
